src/app/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
+file(GLOB QBT_TS_FILES "${qBittorrent_SOURCE_DIR}/src/lang/*.ts")
+# Custom target to update .ts files and include new strings from source files
+# Depends on Qt's LinguistTools
+get_target_property(QT_LUPDATE_EXECUTABLE Qt::lupdate IMPORTED_LOCATION)
+add_custom_target(qbt_update_translations
+                    COMMAND ${QT_LUPDATE_EXECUTABLE} -extensions ui,c,c++,cc,cpp,cxx,ch,h,h++,hh,hpp,hxx
```

Comment:
Do you really need all these extensions?

Reply:
Maybe consider it future proofing?
I could drop the `-extesions` flag but then it will try to parse the .js files too.

Reply:
I mean do you really need to list all these **exotic** extensions like cc, cxx, ch, h++, etc.?

Reply:
A **need**? Probably not. I just reduced the list of possible extensions down to the ones for C/C++ and .ui. I didn't put too much though in it.